### PR TITLE
Update the exception listener doc

### PR DESCRIPTION
**Pull request by Ener-Getick**
See https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1410

cc @xabbuh

### CODE REVIEW

**Comment by xabbuh** [comment]
@Ener-Getick I am sorry. @jovobe and me missed this when updating #1410. I suggest to merge #1410 as it also adds a test to ensure that the listener behaves as expected and change this to only backport/update the documentation. What do you think?

**Comment by Ener-Getick** [comment]
@xabbuh fine for me ☺

File: Resources/doc/4-exception-controller-support.rst [hunk 1]
```diff
@@ -76,21 +68,9 @@ mapping, you can do this in your controller:
...
-        service:
-            exception_handler:    fos_rest.view.exception_wrapper_handler
-        view:
-            # only relevant when using the JMS serializer for serialization
-            exception_wrapper_handler:  null
-
+use serializer normalizers.
 
-See `this example configuration`_ for more details.
+See `how to create handlers`_ for the jms serializer and `how to create normalizers`_ for the symfony serializer.
```
Comment (xabbuh) [inline Resources/doc/4-exception-controller-support.rst]: I would write "JMS serializer" and "Symfony serializer"

File: Resources/doc/4-exception-controller-support.rst [hunk 2]
```diff
@@ -98,6 +78,7 @@ That was it!
 
     If you are receiving a 500 error where you would expect a different response, the issue
     is likely caused by an exception inside the ExceptionController. For example a template
-    is not found or the serializer failed.
+    is not found or the serializer failed. You should take a look to the logs of your app to see if an uncaughted exception has been logged.
```
Comment (xabbuh) [inline Resources/doc/4-exception-controller-support.rst]: [...] take a look at [...]

Comment (xabbuh) [inline Resources/doc/4-exception-controller-support.rst]: [...] an uncaught exception [...]

**Comment by Ener-Getick** [comment]
Updated @xabbuh

**Comment by xabbuh** [comment]
Thank you @Ener-Getick.

**Pull request merged**